app/exchanges/volume_tracking/message_builder.rb

```diff
@@ -20,7 +20,9 @@ def publish_data # rubocop:disable Metrics/MethodLength
 
       aliquot = object
       data = { source_type: '', source_barcode: '', sample_name: '',
-               used_by_type: 'nil', used_by_barcode: '', aliquot_uuid: aliquot.uuid || '' }
+               used_by_type: 'nil', used_by_barcode: '',
+               aliquot_uuid: SecureRandom.uuid,
```

Comment:
I don't think we can use SecureRandom.uuid here, because it need to be the same as what it is with aliquot object. It should be aliquot.uuid